### PR TITLE
Improve software quality of package model

### PR DESCRIPTION
**Pull request by celinecamacho**
Refactor, comment and test all classes in the package model. 

Packages inside model:
- [x] event
- [x] network.answer
- [x] network.query

Before: 84% class coverage, 79% line coverage
After: 97% class coverage, 91% line coverage

### CODE REVIEW

File: app/src/main/java/com/github/dedis/student20_pop/model/network/method/message/data/lao/StateLao.java [hunk 1]
```diff
@@ -12,32 +12,32 @@
  */
 public class StateLao extends Data {
 
-    private final String id; //Hash (organizer + creation + name)
+    private final String id;
     private final String name;
     private final long creation;
-    private final long last_modified;
+    private final long lastModified;
```
Comment (GabrielFleischer) [inline app/src/main/java/com/github/dedis/student20_pop/model/network/method/message/data/lao/StateLao.java:18]: Problem, gson uses the field name to get the json property name.
So we need to add @SerializeName("last_modified")

File: app/src/main/java/com/github/dedis/student20_pop/model/network/method/message/data/lao/StateLao.java [hunk 2]
```diff
@@ -51,7 +51,7 @@ public long getCreation() {
     }
 
     public long getLast_modified() {
```
Comment (GabrielFleischer) [inline app/src/main/java/com/github/dedis/student20_pop/model/network/method/message/data/lao/StateLao.java:53]: Change this name

File: app/src/main/java/com/github/dedis/student20_pop/model/network/method/message/data/lao/UpdateLao.java [hunk 3]
```diff
@@ -12,58 +12,72 @@
  */
 public class UpdateLao extends Data {
 
+    private final String id;
     private final String name;
-    private final long last_modified;
+    private final long lastModified;
```
Comment (GabrielFleischer) [inline app/src/main/java/com/github/dedis/student20_pop/model/network/method/message/data/lao/UpdateLao.java:17]: Same as StateLao

File: app/src/main/java/com/github/dedis/student20_pop/model/network/method/message/data/meeting/CreateMeeting.java [hunk 4]
```diff
@@ -1,44 +1,45 @@
...
  * Data sent to create a new meeting
  */
 public class CreateMeeting extends Data {
 
-    private final String id; // Hash(lao_id + creation + name)
+    private final String id;
     private final String name;
     private final long creation;
-    private final long last_modified;
+    private final long lastModified;
```
Comment (GabrielFleischer) [inline app/src/main/java/com/github/dedis/student20_pop/model/network/method/message/data/meeting/CreateMeeting.java:15]: Same as StateLao

File: app/src/main/java/com/github/dedis/student20_pop/model/network/method/message/data/meeting/StateMeeting.java [hunk 5]
```diff
@@ -1,42 +1,43 @@
...
  * Data received to track the state of a meeting
  */
 public class StateMeeting extends Data {
 
-    private final String id; // Hash(lao_id + creation + name)
+    private final String id;
     private final String name;
     private final long creation;
-    private final long last_modified;
+    private final long lastModified;
```
Comment (GabrielFleischer) [inline app/src/main/java/com/github/dedis/student20_pop/model/network/method/message/data/meeting/StateMeeting.java:15]: Same as state lao

File: app/src/main/java/com/github/dedis/student20_pop/model/network/method/message/data/message/WitnessMessage.java [hunk 6]
```diff
@@ -1,22 +1,36 @@
...
+import com.github.dedis.student20_pop.model.network.method.message.data.Data;
+import com.github.dedis.student20_pop.model.network.method.message.data.Objects;
 
 /**
  * Data sent to attest the message as a witness
  */
 public class WitnessMessage extends Data {
 
-    private final String message_id;
+    private final String messageId;
```
Comment (GabrielFleischer) [inline app/src/main/java/com/github/dedis/student20_pop/model/network/method/message/data/message/WitnessMessage.java:12]: Same as state lao

File: app/src/main/java/com/github/dedis/student20_pop/model/network/method/message/data/lao/StateLao.java (hunk 1 above)
Comment (GabrielFleischer) [inline app/src/main/java/com/github/dedis/student20_pop/model/network/method/message/data/lao/StateLao.java:18]: This name should not be changed. It is used by Gson to compute the json property name,
You need to add @SerializeName("last_modified")

File: app/src/main/java/com/github/dedis/student20_pop/model/network/method/message/data/lao/StateLao.java (hunk 2 above)
Comment (GabrielFleischer) [inline app/src/main/java/com/github/dedis/student20_pop/model/network/method/message/data/lao/StateLao.java:53]: Change this to fit the name

**Review by GabrielFleischer — CHANGES_REQUESTED** [review]
Bugs were created in with Gson as the fields were changed in 
StateLao
UpdateLao
CreateMeeting
StateMeeting
WitnessMessage

@SerlializeName() msut be added

File: app/src/main/java/com/github/dedis/student20_pop/model/event/Event.java [hunk 7]
```diff
@@ -25,19 +18,16 @@
     private final long time;
     private final String id;
     private final String lao;
-    // Can use GeoLocation in the future
+    private ObservableArrayList<String> attendees;
```
[the remark this answers is not in the document]
Comment (celinecamacho) [reply app/src/main/java/com/github/dedis/student20_pop/model/event/Event.java:20]: Indeed, i'm going to change that

**Review by GabrielFleischer — APPROVED** [review]

**Review by jbmichel14 — APPROVED** [review]

**Review by AlexisTabin — APPROVED** [review]

**Pull request merged**